tests/suites/0_stateless/12_time_travel/12_0005_time_travel_alter_add_column_select_at.sh

```diff
@@ -0,0 +1,37 @@
+#!/usr/bin/env bash
+
+CURDIR=$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)
+. "$CURDIR"/../../../shell_env.sh
+
+
+## Create table t12_0005
+echo "create table t12_0005(c int)" | $MYSQL_CLIENT_CONNECT
```

Comment:
It would be better to move these test cases to `alter table` part, and add update/delete integration check.